xbmc/profiles/ProfileManager.cpp

```diff
@@ -129,8 +129,7 @@ void CProfileManager::OnSettingsLoaded()
 
 void CProfileManager::OnSettingsSaved() const
```

Comment:
There is no need to keep this empty function. It has an empty implementation in `ISettingsHandler`.

Reply:
Removed the override. Thanks.